chainntnfs/txnotifier.go

```diff
@@ -1507,6 +1507,13 @@ func (n *TxNotifier) handleConfDetailsAtTip(confRequest ConfRequest,
 
 	// TODO(wilmer): cancel pending historical rescans if any?
 	confSet := n.confNotifications[confRequest]
+
+	// If we already have details for this request, we don't want to add it
+	// again since we have already dispatched notifications for it.
+	if confSet.details != nil {
```

Comment:
Maybe also update the godoc for the `detail` field as it is becoming more than just a cache at this point